templates/zerver/help/move-content-to-another-topic.md

```diff
@@ -29,7 +29,9 @@ for the details on when topic editing is allowed.
 
 1. From the dropdown menu, select which messages to move.
 
-1. Click **Save**.
+1. Toggle whether automated notices should be sent.
```

Comment:
Do we want this to say "Toggle whether you'd like automated notices to be sent"? The current phrasing doesn't make it clear this is a choice. I also wonder if we should add a sentence or two talking about the tradeoffs and etiquette around automated notices -- for example, it can be helpful to mention a user who may not read messages in the new location after moving a thread they started to a new place if you didn't send a notice to the old thread.

Reply:
Yeah, I think it's a good idea to emphasize that it's a choice, and talk about the tradeoffs and etiquette around automated notices. I'll add it to my list of follow-ups.

Reply:
Works for me!